ddtrace/tracer.py

```diff
@@ -944,6 +945,35 @@ def set_tags(self, tags):
         """
         self._tags.update(tags)
 
+    def set_user(self, user_id, name="", email="", scope="", role="", session_id=""):
```

Comment:
I think this is better suited in the `trace_utils` module, we don't typically add integration specific code to `Tracer` or `Span`

Reply:
@Kyle-Verhoog [other languages added this feature at top level](https://docs.datadoghq.com/security_platform/application_security/setup_and_configure/?tab=set_tag&code-lang=nodejs)

I think this approach could improve the "UX" of the developers who use the package, what's your thoughts? DMing me if you want :)